packages/components/psammead-radio-schedule/src/ProgramCard/index.jsx

```diff
@@ -164,7 +164,7 @@ const renderHeaderContent = ({
             {`${stateLabel.toUpperCase()}`}
           </LabelWrapper>
           <VisuallyHiddenText {...hiddenTextProps}>
```

Comment:
We don't want to have a `<VisuallyHiddenText>,</VisuallyHiddenText>` if the state label is not 'Live', so we should do the check prior to the component.
```
{ stateLabel === 'Live' && <VisuallyHiddenText {...hiddenTextProps}>{` ${stateLabel}, `}</VisuallyHiddenText> }
```

Reply:
I had this condition previously but changed it because of https://github.com/bbc/psammead/pull/3160#discussion_r381312079 . Seems that we need that comma for separating state and brand title even if the Program is not live